### DISCUSSION TITLE
Selfhost mail backup server

### DISCUSSION THREAD

**Discussion by Citroene** [tteck/Proxmox#3743] [Ideas]
Hello everyone,

This may not be the right place to ask but..
I am currently using the free mailbox from google with 15GB storage.
My mailbox is 90% full and I am therefore looking for a solution to move my stored emails to a self hosted mail backup server.

Does anyone have experience with this and know a good self hosted solution?

Thanks in advance!
kind regards

**Comment by Mellowlynx** [comment]
1. clean up.
Mails are text, I have years of mails backed-up, and it's below 5 GB total.
Remove emails with attachments, and just remove everything that is not that imported. (it's less than you think, you really need.)

2. I would setup a simple mail server and use imapsync to get the mail to that machine.
No need to open ports if you let imapsync put it on the localhost.

I have a Windows server VM that does this, and use xampp.
I only run the service when I back up and then turn it off.
